src/services/alert/MessageProvider/status/index.js

```diff
@@ -0,0 +1,92 @@
+const { Configuration } = require('@schul-cloud/commons');
+const api = require('../../../../helper/externalApiRequest');
+const logger = require('../../../../../src/logger');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Useless path segments for "../../../../../src/logger", should be "../../../../logger"](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4499680)